src/main/scala/HangMan.scala

```diff
@@ -1,3 +1,88 @@
-object HangMan extends App{
+import org.w3c.dom.css.Counter
+
+import scala.collection.mutable.ArrayBuffer
+import scala.io.Source
+import scala.io.StdIn.readLine
+
+object HangMan extends App
+{
+  val wordList = loadWordList()
+  val wordToGuess = pickWord(wordList)
+  var wordDisplay: ArrayBuffer[Char] = createWordDisplay(wordToGuess)
+  var hangManGrid = Array.ofDim[String](3, 4)
+
+  println(wordToGuess)
+  println(createWordDisplay(wordToGuess).mkString)
+  while( 1 != 0)
+  {
+    checkInput(wordToGuess ,takeInput(), wordDisplay)
+  }
+  //loadWordList().foreach(item => println(item))
+  //val FILENAME =
+
+
+
+
+
+
+  def pickWord(wordList: List[String]): String =
+  {
+    var rand = scala.util.Random
+    var i = rand.nextInt((wordList.size -1))
+    wordList(i)
+  }
+
+  def takeInput(): String = readLine()
+
+  def checkInput(wordToGuess: String , input: String , wordDisplay: ArrayBuffer[Char] ): Boolean =
+  {
+    val wordToGuessCharArray  = wordToGuess.toCharArray
+    var guess = false
```

Comment:
this could be a val, if you were to assign the value returned from the if else statement